papaparse.js

```diff
@@ -592,7 +601,10 @@ License: MIT
 			if (!this._config.step && !this._config.chunk) {
 				this._completeResults.data = this._completeResults.data.concat(results.data);
 				this._completeResults.errors = this._completeResults.errors.concat(results.errors);
-				this._completeResults.meta = results.meta;
+				this._completeResults.meta = Object.assign({}, results.meta, {
```

Comment:
for complete callback, startByte is the start of all chunk, not just the latest chunk.